stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/viewmodel/messages/MessageListViewModel.kt

```diff
@@ -544,9 +550,10 @@ public class MessageListViewModel(
      * [MessageItemGroupPosition.Bottom] or [MessageItemGroupPosition.None] if the message isn't in a group.
      *
      * @param messages The messages we need to group.
+     * @param isInThread If we are in inside a thread.
      * @return A list of [MessageListItemState]s, each containing a position.
      */
-    private fun groupMessages(messages: List<Message>): List<MessageListItemState> {
+    private fun groupMessages(messages: List<Message>, isInThread: Boolean): List<MessageListItemState> {
```

Comment:
A small fix. Thread and channel messages are observed and combined at the same time, therefore we need to pass this parameter explicitly.